collection_requirements.rst

```diff
@@ -280,7 +280,9 @@ licensing and legal concerns that may otherwise affect your code.
 Repository management
 =====================
 
-Every collection MUST have a public SCM repository and releases of the collection MUST be tagged in this repository.
+Every collection MUST have a public git repository and releases of the collection MUST be tagged in this repository.
+
+We are open to add more SCMs once our tooling supports them.
```

Comment:
If we're explicitly requiring a git repository, here is how I'd reword my original suggestion. If there's consensus to handle this separately, I can create a separate PR.

```suggestion
Every collection MUST have a public git repository. Releases of the collection MUST be tagged in said repository. This means that releases MUST be ``git tag``ed and that the tag name MUST exactly match the Galaxy version number. Tag names MAY have a ``v`` prefix, but a collection's tag names MUST have a consistent format from release to release.

Additionally, collection artifacts released to Galaxy MUST be built from the sources that are tagged in the collection's git repository as that release. Any changes made during the build process MUST be clearly documented so the collection artifact can be reproduced.

We are open to allowing other SCM software once our tooling supports them.
```